charts/project-config-connector/templates/project.yaml

```diff
@@ -42,4 +42,28 @@ spec:
   {{- if .Values.resourceID }}
   resourceID: {{ .Values.resourceID | quote }}
   {{- end }}
+
+{{- if $.Values.services }}
+{{- range $service := .Values.services.list  }}
+---
+apiVersion: serviceusage.cnrm.cloud.google.com/v1beta1
+kind: Service
+metadata:
+  annotations:
+    {{- if $.Values.services.protection }}
+    cnrm.cloud.google.com/deletion-policy: {{ $.Values.services.protection.deletionPolicy | default "abandon"  | quote }}
```

Comment:
i'm not sure about defaulting deletion policy to abandon, as using this as replacement to standard iac, i would expect to this be deleted by default, and change this to be deleted? Especialy with non data related resources? i would accept this in such think a bucket / cloudsql instance, as this also works similar in terraform, but with api?

Reply:
for me, it is 50/50. lack of abandon is risky and can leads to accidental downtime of some services. in 90% cases when you enable some API, you forget about it